PennMobile/General/NotificationRequestable.swift

```diff
@@ -60,25 +72,24 @@ extension NotificationRequestable where Self: UIViewController {
             if UIApplication.shared.canOpenURL(settingsUrl) {
                 UIApplication.shared.open(settingsUrl, completionHandler: { (success) in
                     // Checking for setting is opened or not
-                    print("Setting is opened: \(success)")
                     DispatchQueue.main.async {
                         completion?(success)
                     }
                 })
             }
         }
-        alertView.addButton("Decline", action: {  })
+        alertView.addButton("Decline", backgroundColor: UIColor(r: 114, g: 115, b: 117), action: {  })
```

Comment:
Use a real color instead of hardcoding values pls

Reply:
Implemented.